domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/BeanValidator.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.cmc.domain.constraints;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
+
+public class BeanValidator {
```

Comment:
This is copied over from the test folder. I didn't want to move this class now to avoid bloating this PR. Will cleanup in a followup PR.